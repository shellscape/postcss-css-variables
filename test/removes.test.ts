import { fixture, process } from './helpers';

const tests = {
  'should clean up neseted rules if we removed variable declarations to make it empty':
    'remove-nested-empty-rules-after-variable-collection',
  'should clean up rules if we removed variable declarations to make it empty':
    'remove-empty-rules-after-variable-collection'
};

describe('removes', async () => {
  test.each(Object.entries(tests))('%s', async (_desc, file) => {
    const css = await fixture(`removes/${file}`);
    const result = await process(css);
    expect(result).toMatchSnapshot();
  });
});
