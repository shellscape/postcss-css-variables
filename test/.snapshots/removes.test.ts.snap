// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`removes > should clean up neseted rules if we removed variable declarations to make it empty 1`] = `"@media (max-width: 1000px){@media (max-height: 500px){:root{background:#ff0000}}}"`;

exports[`removes > should clean up rules if we removed variable declarations to make it empty 1`] = `":root{background:#ff0000}"`;
